2023-09-17 12:38:35,628 ERROR    <module>  | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 12:44:19,096 ERROR    <module>  | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 12:47:24,243 ERROR    <module>  | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 12:50:08,683 ERROR    <module>  | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 12:51:23,012 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:00:21,140 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:00:21,141 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:00:21,141 INFO     _log  * Restarting with stat
2023-09-17 13:00:21,881 WARNING  _log  * Debugger is active!
2023-09-17 13:00:21,889 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:00:21,921 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:21] "GET / HTTP/1.1" 200 -
2023-09-17 13:00:22,007 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:22] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:00:22,090 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:22] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:26,012 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:26] "GET / HTTP/1.1" 200 -
2023-09-17 13:00:26,024 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:26] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:00:26,248 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:26] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:26,924 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:26] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:00:27,140 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:27] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:00:28,338 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:28] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:00:28,624 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:28] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:28,657 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:28] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:28,665 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:28] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:28,665 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:28] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:00:32,071 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:00:32,123 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:32] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:00:32,224 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:00:32] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:01:29,522 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:29] "GET / HTTP/1.1" 200 -
2023-09-17 13:01:29,823 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:29] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:01:31,213 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:31] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:01:31,457 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:31] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:01:32,479 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:32] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:01:32,808 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:32] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:01:32,808 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:32] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:01:32,822 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:32] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:01:32,826 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:32] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:01:36,441 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:01:36,479 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:36] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:01:36,581 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:36] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:01:39,493 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:01:39] "GET /static/logFiles/server_messages.log HTTP/1.1" 200 -
2023-09-17 13:02:02,527 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:02:02] "GET /static/logFiles/my_messages.txt HTTP/1.1" 200 -
2023-09-17 13:03:44,300 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:03:44] "GET / HTTP/1.1" 200 -
2023-09-17 13:03:44,322 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:03:44] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:03:46,073 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:03:46] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:03:46,380 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:03:46] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:04:00,159 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:00] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:04:00,410 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:00] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:04:00,430 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:00] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:04:00,491 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:00] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:04:00,499 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:00] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:04:03,801 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:04:03,845 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:03] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:04:04,272 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:04] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:04:12,177 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:12] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:04:43,208 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:04:43] "GET /static/logFiles/my_messages.txt HTTP/1.1" 200 -
2023-09-17 13:05:59,525 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:05:59] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:05:59,545 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:05:59] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:06:01,192 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:01] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:06:01,433 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:01] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:06:01,460 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:01] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:06:01,523 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:01] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:06:01,523 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:01] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:06:04,960 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:06:04,996 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:04] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:06:05,373 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:06:05] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:07:25,354 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:25] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:07:25,362 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:25] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:25,672 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:25] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:25,687 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:25] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:25,689 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:25] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:26,012 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:26] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:26,900 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:26] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:07:27,213 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:27] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:07:28,290 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:28] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:07:28,522 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:28] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:28,559 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:28] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:28,621 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:28] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:28,621 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:28] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:07:32,149 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:07:32,190 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:32] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:07:32,558 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:07:32] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:09:19,542 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:19] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:09:19,549 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:19] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:09:19,875 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:19] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:09:19,879 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:19] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:09:19,883 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:19] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:09:20,204 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:20] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:09:23,177 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:23] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:09:23,545 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:23] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:09:34,859 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:09:34] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:10:54,661 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:54] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:10:54,672 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:54] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:10:54,979 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:54] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:10:54,992 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:54] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:10:54,992 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:54] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:10:55,316 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:55] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:10:58,897 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:10:58,939 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:58] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:10:59,308 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:10:59] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:11:31,952 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:31] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:11:31,972 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:31] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:11:32,275 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:32] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:11:32,289 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:32] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:11:32,293 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:32] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:11:32,605 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:32] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:11:35,469 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:11:35,498 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:35] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:11:35,874 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:11:35] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:12:00,347 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:00] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:12:00,357 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:00] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:00,674 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:00] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:00,680 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:00] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:00,680 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:00] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:01,015 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:01] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:03,969 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 58, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:12:04,005 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:04] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:12:04,441 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:04] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:12:41,683 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:41] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:12:41,699 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:41] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:42,025 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:42] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:42,027 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:42] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:42,031 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:42] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:42,358 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:42] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:12:45,872 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:12:45,914 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:45] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:12:46,278 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:12:46] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:13:03,719 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:03] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:13:50,329 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:13:50,455 INFO     _log  * Restarting with stat
2023-09-17 13:13:51,358 WARNING  _log  * Debugger is active!
2023-09-17 13:13:51,372 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:13:56,943 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:56] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:13:57,056 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:57] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:13:57,280 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:57] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:13:57,280 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:57] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:13:57,290 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:57] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:13:57,613 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:13:57] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:00,450 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 42, in write_log_file
    f = open(logFile_path, "a")
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlogstatic/logFiles/my_messages.txt'
2023-09-17 13:14:00,627 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 42, in write_log_file
    f = open(logFile_path, "a")
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlogstatic/logFiles/my_messages.txt'
2023-09-17 13:14:00,627 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 42, in write_log_file
    f = open(logFile_path, "a")
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlogstatic/logFiles/my_messages.txt'
2023-09-17 13:14:00,627 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:14:00,672 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:00] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:14:00,996 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 42, in write_log_file
    f = open(logFile_path, "a")
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlogstatic/logFiles/my_messages.txt'
2023-09-17 13:14:01,027 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:01] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:14:10,243 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:14:10,372 INFO     _log  * Restarting with stat
2023-09-17 13:14:11,141 WARNING  _log  * Debugger is active!
2023-09-17 13:14:11,141 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:14:19,442 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:14:19,589 INFO     _log  * Restarting with stat
2023-09-17 13:14:20,544 WARNING  _log  * Debugger is active!
2023-09-17 13:14:20,556 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:14:25,921 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:25] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:14:26,033 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:26] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:26,257 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:26] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:26,257 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:26] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:26,257 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:26] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:26,609 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:26] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:14:29,284 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 40, in write_log_file
    logFile_path = basedir + "/static/logFiles/" + lgFile
NameError: name 'lgFile' is not defined
2023-09-17 13:14:29,439 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 40, in write_log_file
    logFile_path = basedir + "/static/logFiles/" + lgFile
NameError: name 'lgFile' is not defined
2023-09-17 13:14:29,439 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 40, in write_log_file
    logFile_path = basedir + "/static/logFiles/" + lgFile
NameError: name 'lgFile' is not defined
2023-09-17 13:14:29,439 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:14:29,496 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:29] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:14:29,823 ERROR    write_log_file  | exception from 'write_log_file()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 40, in write_log_file
    logFile_path = basedir + "/static/logFiles/" + lgFile
NameError: name 'lgFile' is not defined
2023-09-17 13:14:29,858 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:29] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:14:41,412 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:41] "GET /static/logFiles/my_messages.txt HTTP/1.1" 200 -
2023-09-17 13:14:54,438 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:14:54] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:16:24,965 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:16:24] "[36mGET /static/logFiles/my_messages.txt HTTP/1.1[0m" 304 -
2023-09-17 13:16:50,547 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:16:50,633 INFO     _log  * Restarting with stat
2023-09-17 13:16:51,324 WARNING  _log  * Debugger is active!
2023-09-17 13:16:51,324 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:17:51,875 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:17:51,975 INFO     _log  * Restarting with stat
2023-09-17 13:17:52,810 WARNING  _log  * Debugger is active!
2023-09-17 13:17:52,818 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:17:55,975 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:17:56,123 INFO     _log  * Restarting with stat
2023-09-17 13:17:57,091 WARNING  _log  * Debugger is active!
2023-09-17 13:17:57,091 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:00,226 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:18:00,377 INFO     _log  * Restarting with stat
2023-09-17 13:18:01,221 WARNING  _log  * Debugger is active!
2023-09-17 13:18:01,229 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:04,430 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:18:04,531 INFO     _log  * Restarting with stat
2023-09-17 13:18:05,207 WARNING  _log  * Debugger is active!
2023-09-17 13:18:05,221 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:07,350 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:18:07,479 INFO     _log  * Restarting with stat
2023-09-17 13:18:08,207 WARNING  _log  * Debugger is active!
2023-09-17 13:18:08,207 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:23,771 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:18:23,884 INFO     _log  * Restarting with stat
2023-09-17 13:18:24,723 WARNING  _log  * Debugger is active!
2023-09-17 13:18:24,723 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:27,872 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:18:27,983 INFO     _log  * Restarting with stat
2023-09-17 13:18:41,756 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:18:41,756 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:18:41,756 INFO     _log  * Restarting with stat
2023-09-17 13:18:42,429 WARNING  _log  * Debugger is active!
2023-09-17 13:18:42,438 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:18:45,521 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:45] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:18:45,638 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:45] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:45,740 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:45] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:45,876 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:45] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:45,882 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:45] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:46,222 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:46] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:47,422 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:18:47,426 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:47] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:18:47,732 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:47] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:18:49,173 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:49] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:18:49,404 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:49] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:49,433 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:49] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:49,507 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:49] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:49,507 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:49] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:18:52,645 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:18:52,813 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:18:52,813 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:18:52,821 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:18:52,860 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:52] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:18:53,189 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:18:53,228 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:53] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:18:57,893 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:18:57] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:19:21,243 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:19:21,338 INFO     _log  * Restarting with stat
2023-09-17 13:19:22,090 WARNING  _log  * Debugger is active!
2023-09-17 13:19:22,090 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:19:23,167 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:19:23,298 INFO     _log  * Restarting with stat
2023-09-17 13:19:24,032 WARNING  _log  * Debugger is active!
2023-09-17 13:19:24,038 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:19:26,194 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:19:26,337 INFO     _log  * Restarting with stat
2023-09-17 13:19:27,239 WARNING  _log  * Debugger is active!
2023-09-17 13:19:27,249 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:19:49,110 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:19:49,225 INFO     _log  * Restarting with stat
2023-09-17 13:19:49,923 WARNING  _log  * Debugger is active!
2023-09-17 13:19:49,923 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:04,488 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:04,586 INFO     _log  * Restarting with stat
2023-09-17 13:20:05,223 WARNING  _log  * Debugger is active!
2023-09-17 13:20:05,238 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:09,389 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:09,487 INFO     _log  * Restarting with stat
2023-09-17 13:20:10,204 WARNING  _log  * Debugger is active!
2023-09-17 13:20:10,207 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:22,687 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:22,796 INFO     _log  * Restarting with stat
2023-09-17 13:20:23,541 WARNING  _log  * Debugger is active!
2023-09-17 13:20:23,541 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:24,609 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:24,699 INFO     _log  * Restarting with stat
2023-09-17 13:20:25,421 WARNING  _log  * Debugger is active!
2023-09-17 13:20:25,421 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:26,519 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:26,655 INFO     _log  * Restarting with stat
2023-09-17 13:20:27,406 WARNING  _log  * Debugger is active!
2023-09-17 13:20:27,406 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:28,495 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:28,604 INFO     _log  * Restarting with stat
2023-09-17 13:20:29,257 WARNING  _log  * Debugger is active!
2023-09-17 13:20:29,257 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:36,536 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:36,683 INFO     _log  * Restarting with stat
2023-09-17 13:20:37,722 WARNING  _log  * Debugger is active!
2023-09-17 13:20:37,724 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:20:41,895 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:20:41,974 INFO     _log  * Restarting with stat
2023-09-17 13:21:49,500 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:21:49,500 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:21:49,500 INFO     _log  * Restarting with stat
2023-09-17 13:21:50,252 WARNING  _log  * Debugger is active!
2023-09-17 13:21:50,257 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:21:54,088 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:54] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:21:54,463 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:54] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:21:54,673 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:54] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:21:54,675 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:54] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:21:54,675 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:54] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:21:55,009 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:55] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:21:55,624 ERROR    excercises can only concatenate str (not "set") to str
2023-09-17 13:21:55,658 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:55] "[35m[1mGET /excercises HTTP/1.1[0m" 500 -
2023-09-17 13:21:55,935 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:55] "GET /excercises?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2023-09-17 13:21:55,992 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:55] "GET /excercises?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2023-09-17 13:21:56,364 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:56] "GET /excercises?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2023-09-17 13:21:56,637 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:21:56] "[36mGET /excercises?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:09,156 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:22:09,240 INFO     _log  * Restarting with stat
2023-09-17 13:22:20,312 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:22:20,312 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:22:20,312 INFO     _log  * Restarting with stat
2023-09-17 13:22:21,113 WARNING  _log  * Debugger is active!
2023-09-17 13:22:21,113 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:22:26,205 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:26] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:28,422 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:28] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:22:28,689 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:28] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:28,765 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:28] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:28,765 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:28] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:28,772 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:28] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:29,090 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:29] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:22:31,739 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:22:31] "[35m[1mGET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1[0m" 500 -
2023-09-17 13:23:32,036 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:23:32,141 INFO     _log  * Restarting with stat
2023-09-17 13:23:32,822 WARNING  _log  * Debugger is active!
2023-09-17 13:23:32,830 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:23:35,945 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:23:36,030 INFO     _log  * Restarting with stat
2023-09-17 13:23:36,957 WARNING  _log  * Debugger is active!
2023-09-17 13:23:36,957 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:23:45,673 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:45] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:23:45,776 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:45] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:23:46,010 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:46] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:23:46,019 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:46] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:23:46,019 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:46] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:23:46,338 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:46] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:23:49,224 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:23:49,386 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:23:49,386 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:23:49,386 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:23:49,444 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:49] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:23:49,774 ERROR    write_log_file  | exception from 'write_log_file()': module 'datetime' has no attribute 'dtetime' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:23:49,807 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:49] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:23:53,984 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:23:53] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:24:24,673 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:24:24] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:25:15,509 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:25:15,612 INFO     _log  * Restarting with stat
2023-09-17 13:25:16,356 WARNING  _log  * Debugger is active!
2023-09-17 13:25:16,356 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:25:18,467 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:25:18,546 INFO     _log  * Restarting with stat
2023-09-17 13:25:19,375 WARNING  _log  * Debugger is active!
2023-09-17 13:25:19,375 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:25:22,852 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:22] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:25:22,953 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:22] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:25:23,176 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:23] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:25:23,176 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:23] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:25:23,176 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:23] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:25:23,521 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:23] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:25:26,759 ERROR    write_log_file module 'datetime' has no attribute 'dtetime' | exception from 'write_log_file()' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:25:26,912 ERROR    write_log_file module 'datetime' has no attribute 'dtetime' | exception from 'write_log_file()' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:25:26,912 ERROR    write_log_file module 'datetime' has no attribute 'dtetime' | exception from 'write_log_file()' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:25:26,912 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:25:26,960 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:26] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:25:27,292 ERROR    write_log_file module 'datetime' has no attribute 'dtetime' | exception from 'write_log_file()' 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\MyFunc.py", line 45, in write_log_file
    f.write('%s | %s\n' % (datetime.dtetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')[:-3],msg))
AttributeError: module 'datetime' has no attribute 'dtetime'
2023-09-17 13:25:27,321 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:27] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:25:31,762 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:25:31] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:26:07,630 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:26:07,715 INFO     _log  * Restarting with stat
2023-09-17 13:26:08,521 WARNING  _log  * Debugger is active!
2023-09-17 13:26:08,529 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:26:21,987 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:26:22,115 INFO     _log  * Restarting with stat
2023-09-17 13:26:22,922 WARNING  _log  * Debugger is active!
2023-09-17 13:26:22,924 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:26:28,163 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:26:28,293 INFO     _log  * Restarting with stat
2023-09-17 13:26:29,291 WARNING  _log  * Debugger is active!
2023-09-17 13:26:29,304 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:26:32,042 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:26:32,149 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:26:32,397 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:26:32,404 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:26:32,407 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:26:32,722 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:32] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:26:35,744 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 60, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:26:35,805 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:35] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:26:36,161 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:36] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:26:38,627 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:26:38] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:27:11,706 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:11] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:27:11,710 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:11] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:27:12,023 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:12] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:27:12,032 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:12] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:27:12,041 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:12] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:27:12,353 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:12] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:27:15,299 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:15] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:27:15,662 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:27:15] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:28:04,309 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:28:04,328 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:04,645 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:04,659 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:04,660 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:04,982 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:04] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:08,831 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:08] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:28:09,190 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:09] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:28:37,240 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:28:37,262 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:37,574 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:37,574 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:37,589 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:37,909 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:37] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:28:41,060 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:41] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:28:41,410 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:28:41] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:29:29,697 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:29] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:29:29,704 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:29] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:29:30,024 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:30] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:29:30,033 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:30] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:29:30,037 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:30] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:29:30,355 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:30] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:29:33,722 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:33] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:29:34,078 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:29:34] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:34:42,322 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:34:42,444 INFO     _log  * Restarting with stat
2023-09-17 13:34:43,139 WARNING  _log  * Debugger is active!
2023-09-17 13:34:43,139 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:34:44,194 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:34:44,266 INFO     _log  * Restarting with stat
2023-09-17 13:34:44,907 WARNING  _log  * Debugger is active!
2023-09-17 13:34:44,907 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:35:57,478 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:35:57,598 INFO     _log  * Restarting with stat
2023-09-17 13:35:58,340 WARNING  _log  * Debugger is active!
2023-09-17 13:35:58,340 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:36:06,672 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:36:06,801 INFO     _log  * Restarting with stat
2023-09-17 13:36:07,460 WARNING  _log  * Debugger is active!
2023-09-17 13:36:07,460 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:36:08,537 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:36:08,624 INFO     _log  * Restarting with stat
2023-09-17 13:36:09,322 WARNING  _log  * Debugger is active!
2023-09-17 13:36:09,340 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:36:10,408 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:36:10,516 INFO     _log  * Restarting with stat
2023-09-17 13:36:11,240 WARNING  _log  * Debugger is active!
2023-09-17 13:36:11,240 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:36:18,570 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:36:18,675 INFO     _log  * Restarting with stat
2023-09-17 13:39:15,554 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:39:15,554 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:39:15,554 INFO     _log  * Restarting with stat
2023-09-17 13:39:16,119 WARNING  _log  * Debugger is active!
2023-09-17 13:39:16,119 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:39:21,929 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:21] "GET / HTTP/1.1" 200 -
2023-09-17 13:39:22,037 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:22] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:39:23,604 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:23] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:39:23,907 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:23] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:39:24,924 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:24] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:39:25,137 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:25] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:25,175 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:25] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:25,272 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:25] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:25,272 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:25] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:28,872 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:28] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:39:29,223 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:29] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:39:47,131 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:39:47,146 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:47,463 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:47,465 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:47,469 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:47,806 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:47] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:39:51,237 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 65, in <module>
    prRed(ms)
NameError: name 'ms' is not defined
2023-09-17 13:39:51,277 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:51] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:39:51,642 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:51] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:39:55,994 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:39:55] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:40:06,316 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:40:06] "GET /static/logFiles/my_messages.txt HTTP/1.1" 200 -
2023-09-17 13:40:41,877 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:40:41,977 INFO     _log  * Restarting with stat
2023-09-17 13:40:42,723 WARNING  _log  * Debugger is active!
2023-09-17 13:40:42,723 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:40:44,831 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:40:44,918 INFO     _log  * Restarting with stat
2023-09-17 13:40:45,587 WARNING  _log  * Debugger is active!
2023-09-17 13:40:45,587 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:40:49,771 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:40:49,872 INFO     _log  * Restarting with stat
2023-09-17 13:40:50,590 WARNING  _log  * Debugger is active!
2023-09-17 13:40:50,590 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:40:53,717 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:40:53,823 INFO     _log  * Restarting with stat
2023-09-17 13:40:54,557 WARNING  _log  * Debugger is active!
2023-09-17 13:40:54,572 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:40:58,736 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\config_logging.py', reloading
2023-09-17 13:40:58,831 INFO     _log  * Restarting with stat
2023-09-17 13:40:59,507 WARNING  _log  * Debugger is active!
2023-09-17 13:40:59,507 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:42:44,382 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\MyFunc.py', reloading
2023-09-17 13:42:44,495 INFO     _log  * Restarting with stat
2023-09-17 13:42:45,239 WARNING  _log  * Debugger is active!
2023-09-17 13:42:45,239 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:42:58,590 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:42:58,590 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:42:58,603 INFO     _log  * Restarting with stat
2023-09-17 13:42:59,360 WARNING  _log  * Debugger is active!
2023-09-17 13:42:59,360 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:43:13,170 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:13] "GET / HTTP/1.1" 200 -
2023-09-17 13:43:13,270 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:13] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:43:14,631 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:14] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:43:14,944 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:14] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:43:16,270 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:16] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:43:16,483 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:16] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:16,520 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:16] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:16,610 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:16] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:16,620 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:16] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:20,278 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:20] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:43:20,643 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:20] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:43:43,702 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:43] "GET /excercise/?id=1 HTTP/1.1" 200 -
2023-09-17 13:43:43,720 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:43] "[36mGET /static/img/icons8-script-68.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:44,023 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:44] "[36mGET /static/img/icons8-descargar-de-la-nube-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:44,034 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:44] "[36mGET /static/img/icons8-html-filetype-50.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:44,037 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:44] "[36mGET /static/img/icons8-consola-100.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:44,361 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:44] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:43:48,125 ERROR    <module> name 'ms' is not defined | exception from '0-prototype-colors.py()': 
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\static\py_excercises\20230213-PrototypeScript-colors\0-prototype-colors1.py", line 59, in <module>
    print(f"\t{ms}")
NameError: name 'ms' is not defined
2023-09-17 13:43:48,156 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:48] "GET /result_script_exec/?py_path=static/py_excercises/20230213-PrototypeScript-colors/0-prototype-colors1.py HTTP/1.1" 200 -
2023-09-17 13:43:48,507 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:43:48] "GET /result_script_html HTTP/1.1" 200 -
2023-09-17 13:50:25,835 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:50:25,934 INFO     _log  * Restarting with stat
2023-09-17 13:50:26,825 WARNING  _log  * Debugger is active!
2023-09-17 13:50:26,837 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:50:30,012 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:50:30,114 INFO     _log  * Restarting with stat
2023-09-17 13:50:30,923 WARNING  _log  * Debugger is active!
2023-09-17 13:50:30,923 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:50:36,920 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:50:36] "GET / HTTP/1.1" 200 -
2023-09-17 13:50:37,039 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:50:37] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:50:38,916 ERROR    excercises excrcises.html
2023-09-17 13:50:38,916 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:50:38,918 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:50:38] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:50:39,220 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:50:39] "GET /static/img/sad-face-2692.png HTTP/1.1" 200 -
2023-09-17 13:52:12,508 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:12] "GET / HTTP/1.1" 200 -
2023-09-17 13:52:12,512 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:12] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:52:12,885 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:12] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:52:13,931 ERROR    excercises excrcises.html
2023-09-17 13:52:13,931 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:52:13,938 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:13] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:52:14,236 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:14] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:52:16,271 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:16] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 13:52:44,248 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:52:44] "GET /static/logFiles/my_messages.txt HTTP/1.1" 200 -
2023-09-17 13:53:13,650 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:13] "GET / HTTP/1.1" 200 -
2023-09-17 13:53:13,668 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:13] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:53:14,034 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:14] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:53:14,740 ERROR    excercises excrcises.html
2023-09-17 13:53:14,745 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:53:14,761 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:14] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:53:15,062 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:15] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:53:35,127 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:35] "GET / HTTP/1.1" 200 -
2023-09-17 13:53:35,130 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:35] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:53:35,490 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:35] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:53:36,369 ERROR    excercises excrcises.html
2023-09-17 13:53:36,371 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:53:36,380 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:36] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:53:36,673 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:53:36] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:54:12,871 ERROR    excercises excrcises.html
2023-09-17 13:54:12,871 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:54:12,871 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:12] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:54:12,886 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:12] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:54:13,137 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:13] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:54:30,728 ERROR    excercises excrcises.html
2023-09-17 13:54:30,731 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:54:30,754 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:30] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:54:30,755 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:30] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:54:30,995 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:54:30] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:55:01,187 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:55:01,324 INFO     _log  * Restarting with stat
2023-09-17 13:55:02,039 WARNING  _log  * Debugger is active!
2023-09-17 13:55:02,039 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:55:04,121 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:55:04,212 INFO     _log  * Restarting with stat
2023-09-17 13:55:05,075 WARNING  _log  * Debugger is active!
2023-09-17 13:55:05,075 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:55:08,278 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:55:08,431 INFO     _log  * Restarting with stat
2023-09-17 13:55:47,926 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:55:47,926 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:55:47,926 INFO     _log  * Restarting with stat
2023-09-17 13:55:48,895 WARNING  _log  * Debugger is active!
2023-09-17 13:55:48,903 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:55:48,939 ERROR    excercises excrcises.html
2023-09-17 13:55:48,940 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:55:48,956 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:48] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:55:49,089 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:49] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:55:49,144 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:49] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:55:51,895 ERROR    excercises excrcises.html
2023-09-17 13:55:51,895 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:55:51,895 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:51] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:55:51,908 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:51] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:55:52,174 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:55:52] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:56:11,921 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:56:12,019 INFO     _log  * Restarting with stat
2023-09-17 13:56:25,900 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:56:25,900 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:56:25,904 INFO     _log  * Restarting with stat
2023-09-17 13:56:26,537 WARNING  _log  * Debugger is active!
2023-09-17 13:56:26,553 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:56:30,303 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:56:30] "GET / HTTP/1.1" 200 -
2023-09-17 13:56:30,670 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:56:30] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:56:31,004 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:56:31] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:56:31,602 ERROR    excercises excrcises.html
2023-09-17 13:56:31,604 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:56:31,606 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:56:31] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:56:31,903 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:56:31] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:57:29,003 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:57:29,122 INFO     _log  * Restarting with stat
2023-09-17 13:57:37,011 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:57:37,011 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:57:37,011 INFO     _log  * Restarting with stat
2023-09-17 13:57:37,603 WARNING  _log  * Debugger is active!
2023-09-17 13:57:37,603 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:57:42,207 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:57:42] "GET / HTTP/1.1" 200 -
2023-09-17 13:57:42,315 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:57:42] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:57:42,636 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:57:42] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:57:43,564 ERROR    excercises excrcises.html
2023-09-17 13:57:43,565 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:57:43,570 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:57:43,585 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:57:43] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:57:43,873 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:57:43] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:57:53,267 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:57:53,380 INFO     _log  * Restarting with stat
2023-09-17 13:57:54,153 WARNING  _log  * Debugger is active!
2023-09-17 13:57:54,160 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:57:56,271 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:57:56,392 INFO     _log  * Restarting with stat
2023-09-17 13:57:57,112 WARNING  _log  * Debugger is active!
2023-09-17 13:57:57,119 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:58:01,311 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:58:01,408 INFO     _log  * Restarting with stat
2023-09-17 13:58:19,243 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:58:19,243 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:58:19,243 INFO     _log  * Restarting with stat
2023-09-17 13:58:19,838 WARNING  _log  * Debugger is active!
2023-09-17 13:58:19,838 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:58:24,211 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:24] "GET / HTTP/1.1" 200 -
2023-09-17 13:58:24,315 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:24] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:58:24,638 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:24] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:58:25,634 ERROR    excercises excrcises.html
2023-09-17 13:58:25,635 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:58:25,635 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:58:25,635 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:25] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:58:25,937 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:25] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:58:37,555 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:58:37,667 INFO     _log  * Restarting with stat
2023-09-17 13:58:48,100 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:58:48,100 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:58:48,100 INFO     _log  * Restarting with stat
2023-09-17 13:58:48,673 WARNING  _log  * Debugger is active!
2023-09-17 13:58:48,681 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:58:52,872 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:52] "GET / HTTP/1.1" 200 -
2023-09-17 13:58:52,983 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:52] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:58:53,303 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:53] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:58:53,971 ERROR    excercises excrcises.html
2023-09-17 13:58:53,972 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:58:54,022 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:54] "[35m[1mGET /excercises HTTP/1.1[0m" 500 -
2023-09-17 13:58:54,283 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:54] "[36mGET /excercises?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2023-09-17 13:58:54,347 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:54] "[36mGET /excercises?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2023-09-17 13:58:54,722 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:54] "[36mGET /excercises?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-09-17 13:58:54,986 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:58:54] "[36mGET /excercises?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:11,623 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:59:11,720 INFO     _log  * Restarting with stat
2023-09-17 13:59:27,771 INFO     _log [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-09-17 13:59:27,771 INFO     _log [33mPress CTRL+C to quit[0m
2023-09-17 13:59:27,771 INFO     _log  * Restarting with stat
2023-09-17 13:59:28,788 WARNING  _log  * Debugger is active!
2023-09-17 13:59:28,788 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:59:28,838 ERROR    excercises excrcises.html
2023-09-17 13:59:28,840 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:59:28,848 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:28] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:59:28,923 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:28] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:29,014 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:29] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:32,671 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:32] "GET / HTTP/1.1" 200 -
2023-09-17 13:59:32,743 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:32] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:34,131 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:34] "GET / HTTP/1.1" 200 -
2023-09-17 13:59:34,437 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:34] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 13:59:34,751 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:34] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:35,745 ERROR    excercises excrcises.html
2023-09-17 13:59:35,746 ERROR    excercises exception => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 13:59:35,746 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:35] "GET /excercises HTTP/1.1" 200 -
2023-09-17 13:59:36,061 INFO     _log 127.0.0.1 - - [17/Sep/2023 13:59:36] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 13:59:48,785 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:59:48,885 INFO     _log  * Restarting with stat
2023-09-17 13:59:49,586 WARNING  _log  * Debugger is active!
2023-09-17 13:59:49,594 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 13:59:51,719 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 13:59:51,849 INFO     _log  * Restarting with stat
2023-09-17 13:59:52,506 WARNING  _log  * Debugger is active!
2023-09-17 13:59:52,506 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:01:15,486 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:01:15] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 14:02:42,371 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:02:42,482 INFO     _log  * Restarting with stat
2023-09-17 14:02:43,256 WARNING  _log  * Debugger is active!
2023-09-17 14:02:43,256 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:02:46,415 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:02:46,519 INFO     _log  * Restarting with stat
2023-09-17 14:02:47,257 WARNING  _log  * Debugger is active!
2023-09-17 14:02:47,257 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:02:49,353 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:02:49,474 INFO     _log  * Restarting with stat
2023-09-17 14:02:50,288 WARNING  _log  * Debugger is active!
2023-09-17 14:02:50,288 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:02:52,411 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:02:52,522 INFO     _log  * Restarting with stat
2023-09-17 14:02:53,456 WARNING  _log  * Debugger is active!
2023-09-17 14:02:53,456 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:03:04,917 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:03:05,045 INFO     _log  * Restarting with stat
2023-09-17 14:03:06,054 WARNING  _log  * Debugger is active!
2023-09-17 14:03:06,054 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:03:08,860 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:03:08] "GET / HTTP/1.1" 200 -
2023-09-17 14:03:08,952 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:03:08] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 14:03:09,285 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:03:09] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:03:10,007 ERROR    excercises excrcises.html
2023-09-17 14:03:10,008 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:03:10,010 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:03:10] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:03:10,320 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:03:10] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:02,011 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:02] "GET / HTTP/1.1" 200 -
2023-09-17 14:05:02,028 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:02] "[36mGET /static/img/ame-01.jpg HTTP/1.1[0m" 304 -
2023-09-17 14:05:02,391 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:02] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:03,146 ERROR    excercises excrcises.html
2023-09-17 14:05:03,147 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:05:03,163 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:03] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:05:03,458 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:03] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:14,136 ERROR    excercises excrcises.html
2023-09-17 14:05:14,138 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:05:14,154 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:14] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:05:14,170 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:14] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:14,408 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:14] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:37,906 ERROR    excercises excrcises.html
2023-09-17 14:05:37,906 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:05:37,919 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:37] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:05:37,926 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:37] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:05:38,161 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:05:38] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:06:27,324 ERROR    excercises excrcises.html
2023-09-17 14:06:27,324 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:06:27,352 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:27] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:06:27,367 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:27] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:06:27,578 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:27] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:06:54,753 ERROR    excercises excrcises.html
2023-09-17 14:06:54,756 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:06:54,773 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:54] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:06:54,775 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:54] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:06:55,010 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:06:55] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:07:23,708 ERROR    excercises excrcises.html
2023-09-17 14:07:23,708 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:07:23,722 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:23] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:07:23,737 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:23] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:07:23,963 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:23] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:07:46,886 ERROR    excercises excrcises.html
2023-09-17 14:07:46,888 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:07:46,904 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:46] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:07:46,912 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:46] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:07:47,149 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:07:47] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:08:07,119 ERROR    excercises excrcises.html
2023-09-17 14:08:07,119 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:08:07,135 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:07] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:08:07,153 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:07] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:08:07,377 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:07] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:08:16,632 ERROR    excercises excrcises.html
2023-09-17 14:08:16,632 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:08:16,647 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:16] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:08:16,658 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:16] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:08:16,891 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:08:16] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:08:52,692 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:08:52,802 INFO     _log  * Restarting with stat
2023-09-17 14:08:53,522 WARNING  _log  * Debugger is active!
2023-09-17 14:08:53,522 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:08:54,589 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:08:54,689 INFO     _log  * Restarting with stat
2023-09-17 14:08:55,522 WARNING  _log  * Debugger is active!
2023-09-17 14:08:55,535 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:08:57,621 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:08:57,749 INFO     _log  * Restarting with stat
2023-09-17 14:08:58,540 WARNING  _log  * Debugger is active!
2023-09-17 14:08:58,552 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:09:00,237 ERROR    excercises excrcises.html
2023-09-17 14:09:00,242 ERROR    excercises error from excercises() => excrcises.html
Traceback (most recent call last):
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\flask_app.py", line 46, in excercises
    return render_template('excrcises.html', excercises = Excercises)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 62, in get_source
    return self._get_source_fast(environment, template)
  File "C:\ML_Project\NazarethCourse2023\iggPyBlog\igg_venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: excrcises.html
2023-09-17 14:09:00,259 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:00] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:09:00,315 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:00] "[36mGET /static/img/sad-face-2692.png HTTP/1.1[0m" 304 -
2023-09-17 14:09:00,493 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:00] "[36mGET /static/img/icons8-bulb-64.png HTTP/1.1[0m" 304 -
2023-09-17 14:09:11,368 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:11] "GET /static/logFiles/server_messages.txt HTTP/1.1" 200 -
2023-09-17 14:09:30,968 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:09:31,063 INFO     _log  * Restarting with stat
2023-09-17 14:09:31,905 WARNING  _log  * Debugger is active!
2023-09-17 14:09:31,905 INFO     _log  * Debugger PIN: 139-969-208
2023-09-17 14:09:34,753 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:34] "GET /excercises HTTP/1.1" 200 -
2023-09-17 14:09:34,855 INFO     _log 127.0.0.1 - - [17/Sep/2023 14:09:34] "[36mGET /static/img/Lorea.jpg HTTP/1.1[0m" 304 -
2023-09-17 14:09:48,576 INFO     _log  * Detected change in 'C:\\ML_Project\\NazarethCourse2023\\iggPyBlog\\flask_app.py', reloading
2023-09-17 14:09:48,678 INFO     _log  * Restarting with stat
2023-09-17 14:09:49,323 WARNING  _log  * Debugger is active!
2023-09-17 14:09:49,338 INFO     _log  * Debugger PIN: 139-969-208
